docker/manifest.tmpl

```diff
@@ -0,0 +1,31 @@
+image: drone/drone-fileglob:{{#if build.tag}}{{trimPrefix "v" build.tag}}{{else}}latest{{/if}}
```

Comment:
should be `plugins/drone-fileglob`. I think we need to update the boilr code to match the standards.

Reply:
`plugins` refers to the Docker Hub repository, and `drone-findfiles` is the plugin name

let's also replace the occurrences of fileglob with findfiles in manifest.tmpl, go.mod, main.go, and README.md